### DISCUSSION TITLE
Multiple PNG composite single page TIF, do I want it to be multi layer TIF instead of single layer TIF

### DISCUSSION THREAD

**Discussion by 291598912** [ImageMagick/ImageMagick#6311] [Digital Image Processing]
Multiple PNG composite single page TIF, do I want it to be multi layer TIF instead of single layer TIF

**Comment by snibgo** [comment]
I don't understand the question.

**Comment by 291598912** [comment]
This is the synthesized TIF image. Below are the differences between the two TIF layers：

<img width="704" alt="1" src="https://user-images.githubusercontent.com/16145102/236853396-21847023-f062-4e2c-929a-3439653d99dd.png">
<img width="705" alt="2" src="https://user-images.githubusercontent.com/16145102/236853411-9e4b7f9a-fb35-4f47-a634-197654b79235.png">
